src/components/Chat/MessagesList.tsx

```diff
@@ -5,20 +5,37 @@ import styled from "styled-components";
 import { Message } from "./Message";
 import { WHITE_COLOR } from "../../helpers/styleConstants";
 
-export const MessagesList = ({ messages, userEmail }) => (
-  <MessagesListWrapper>
-    {_.map(messages, message => (
-      <Message key={message._id} idx={message._id} 
-      text={message.text} isMyMessage={message.author.email === userEmail}
-      timestamp={message.timestamp} />
-    ))}
-  </MessagesListWrapper>
-);
-
-const MessagesListWrapper = styled(ScrollView)`
-  flex: 1;
-  padding: 0 20px;
-  display: flex;
-  flex-direction: column;
-  backgroundColor: ${WHITE_COLOR};
-`;
+interface IProps {
+  userEmail: string;
+  messages: object;
+}
+export class MessagesList extends React.Component<IProps> {
+  public scrollToEnd = () => {
+    this.scrollView.scrollToEnd();
+  }
+  public render() {
+    const { messages, userEmail } = this.props
+    return (
+      <ScrollView
+        style={{
```

Comment:
should it be like styled component?

Reply:
No, because ScrollView lose his context, and we can't use refs.

Reply:
ok